src/main/java/gov/cms/madie/models/measure/BaseConfigurationTypes.java

```diff
@@ -13,6 +13,8 @@ public enum BaseConfigurationTypes {
   EFFICIENCY("Efficiency"),
   @JsonProperty("Intermediate Clinical Outcome")
   INTERMEDIATE_CLINICAL_OUTCOME("Intermediate Clinical Outcome"),
```

Comment:
is this for backward compatibility?

Reply:
I believe so. I don't see a reason to remove the jsonProperty. It seems like this pattern was already implemented